### DISCUSSION TITLE
Is PostCSS slow?

### DISCUSSION THREAD

**Discussion by martinszeltins** [tailwindlabs/tailwindcss#8062] [General]
I've heard that PostCSS is slow, so is this an issue for Tailwind CSS?

**Comment by cannap** [comment]
it's faster than the light

**Reply by martinszeltins** [reply]
I thought PostCSS was slow

**Reply by gihrig** [reply]
I have never observed it to be slow. Do you have use case that demonstrates PostCSS being slow?
